src/ChooseArea.svelte

```diff
@@ -20,24 +21,24 @@
 
     const resp = await fetch(authoritiesUrl);
     const body = await resp.text();
-    const json = JSON.parse(body);
+    const json: FeatureCollection = JSON.parse(body);
     for (let feature of json.features) {
       let option = document.createElement("option");
-      option.value = feature.properties.name;
+      option.value = feature.properties!.name;
       dataList.appendChild(option);
-      authoritySet.add(feature.properties.name);
+      authoritySet.add(feature.properties!.name);
     }
 
     // Only show TAs, not LADs, in the map
-    json.features = json.features.filter((f) => f.properties.level == "TA");
+    json.features = json.features.filter((f) => f.properties!.level == "TA");
 
     let map = new maplibregl.Map({
       container: "map",
       style:
         "https://api.maptiler.com/maps/streets/style.json?key=MZEJTanw3WpxRvt7qDfo",
     });
 
-    let hoverId = null;
+    let hoverId: number | null = null;
```

Comment:
We know the IDs are numbers, and not strings, because we use `generateIds: true`